src/server/billing/helpers/adjustUserCount.js

```diff
@@ -121,4 +123,6 @@ export default async function adjustUserCount(userId, orgInput, type) {
   }, []);
 
   await Promise.all(stripePromises);
+  // publish any changes to user traits (like tier counts) to segment:
+  await sendSegmentIdentify(userId);
```

Comment:
No need to await segment things, but it's cool if you wanna keep it

Reply:
Oh true. I'll leave it for now, but if we notice anything laggy we'll know where to start :)